packages/luals-annotations/src/index.ts

```diff
@@ -0,0 +1,47 @@
+import { classes, functions, matchers } from "./json-docs";
+import AnnotationBuilder from "./annotationBuilder";
+import * as fs from "fs";
+
+const document = new AnnotationBuilder();
+document.add("---@meta");
+
+classes.forEach((classDef) => {
+	document.withClass(classDef.name, classDef.description, false, () => {
+		classDef.fields?.forEach((field) => document.addField(field));
+		document.addClassDeclaration();
+		classDef.methods?.forEach((method) => document.addFunction(method));
```

Comment:
I removed the `methods` property before merging the docs. All functions should just be part of `fields`,